app.py

```diff
@@ -114,7 +115,7 @@
         src = img.get_attribute('src')
         urllib.request.urlretrieve(src, "images/"+serial_number_string+".png")
     except TimeoutException:
-        "images/" + serial_number_string + ".png" == "N/A"
+        pass
```

Comment:
If theres a timeout we dont save anything to img/ folder so i just passed the except clause